multi_import/data.py

```diff
@@ -191,28 +194,42 @@ def get_http_response(self, file_format=None, filename=None):
     def _get_format(self, file_format):
         return files.find_format(self.file_formats, file_format)
 
+    def _get_dataset(self, format) -> Dataset:
+        dataset = self.dataset
+        if self.empty and format.empty_file_requires_example_row:
+            dataset = copy(dataset)
+            dataset.append(self.example_row)
+        return dataset
+
 
 class MultiExportResult(object):
+    """
+    A collection of ExportResults
+    """
     def __init__(self, filename, file_formats, results):
         self.file_formats = file_formats
         self.filename = filename
         self.results = results
 
-    def get_file(self, file_format=None):
+    def get_file(self, file_format: FileFormat=None, export_items_as_individual_files: bool=False) -> BytesIO:
         format = self._get_format(file_format)
 
-        if len(self.results) == 1:
+        if self._is_single_content_type_export() and not export_items_as_individual_files:
             return self.results[0].get_file(format)
 
         file = BytesIO()
+
         with zipfile.ZipFile(file, "w") as zf:
             for result in self.results:
-                f = result.get_file(format)
-                fname = "{0}.{1}".format(result.filename, format.extension)
-                zf.writestr(fname, f.getvalue())
+                if export_items_as_individual_files:
+                    self._write_tree_export(format,result, zf)
+                else:
+                    self._write_tabular_export(format, result, zf)
+
         return file
 
-    def get_http_response(self, file_format=None, filename=None):
+    def get_http_response(self, file_format: FileFormat=None, filename: str=None) -> HttpResponse:
```

Comment:
Maybe we should add `export_items_as_individual_files` as a param here too for completeness? Even if we don't need use it yet.